test/parallel/test-os.js

```diff
@@ -93,8 +94,9 @@ const release = os.release();
 is.string(release);
 assert.ok(release.length > 0);
 //TODO: Check format on more than just AIX
-if (common.isAix)
+if (common.isAix) {
```

Comment:
It is still a little confusing to me because it is not obvious when to add braces and when not to. I have seen conditions with one line of code after it being with and without braces. Maybe we need to use some `eslint` rule so it will yell at us on some cases?